src/metrics_server/run_action.sh

```diff
@@ -14,12 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SRC_DIR="src/metrics_server"
-BUILD_DIR="build/metrics_server"
+readonly SRC_DIR="src/metrics_server"
```

Comment:
When should we use `readonly` and when `local -r`? You've replaced the `readonly` in the other file.
Do we really need different syntaxes based on whether it's a function or script-scope variable?